lib/internal/util.js

```diff
@@ -69,16 +69,8 @@ const experimentalWarnings = new SafeSet();
 
 const colorRegExp = /\u001b\[\d\d?m/g; // eslint-disable-line no-control-regex
 
-const unpairedSurrogateRe =
-  /(?:[^\uD800-\uDBFF]|^)[\uDC00-\uDFFF]|[\uD800-\uDBFF](?![\uDC00-\uDFFF])/;
 function toUSVString(val) {
-  const str = `${val}`;
-  // As of V8 5.5, `str.search()` (and `unpairedSurrogateRe[@@search]()`) are
-  // slower than `unpairedSurrogateRe.exec()`.
-  const match = RegExpPrototypeExec(unpairedSurrogateRe, str);
-  if (!match)
-    return str;
-  return _toUSVString(str, match.index);
+  return _toUSVString(`${val}`);
```

Comment:
Small nit it think I had seen using `''+val` rather than template strings leads to small perf bump

Reply:
Ah ok understood! thanks